great_expectations/rule_based_profiler/parameter_builder/parameter_container.py

```diff
@@ -114,6 +115,7 @@ def _convert_dictionaries_to_parameter_nodes(
 
         if isinstance(source, dict):
             if not isinstance(source, ParameterNode):
+                filter_properties_dict(properties=source, inplace=True)
```

Comment:
@anthonyburdi Not false, but specifically NULL/Empty/NonExistent (i.e., valued as None).

Reply:
It looks like `filter_properties_dict` will also filter out falsy values here: https://github.com/great-expectations/great_expectations/blob/aed7c2472a10f7cb0ce2a8429e598f628e2a676f/great_expectations/util.py#L868-L880 which is probably not what we want (i.e. if a user specifies the value of False it'll get removed).

Reply:
@anthonyburdi You mean in line 877?  If so, I will modify this utility function to have the option to not remove the boolean False (only None) -- please confirm that this is what you are seeing.  Thanks!

Reply:
Yes! It looks like line 877 will filter out both False and None and even 0 (since None, 0 is "falsy"). Maybe this utility function should be more explicit and only filter out `None` e.g. `or value is not None`. If there aren't unit tests for this utility function, mind adding a few? To make sure we catch at least this case.

Reply:
This has been done as an upcoming PR to `develop`.

Reply:
@anthonyburdi Might you have a few minutes to review `https://github.com/great-expectations/great_expectations/pull/2878`? Thanks!

Reply:
Thanks @alexsherstinsky - 2878 will fix this issue. Thank you for proposing a fix!